### ISSUE TITLE
FFmpeg extension: Update build instructions to work with newer NDKs

### ISSUE THREAD

**Issue by karma-sun** [google/ExoPlayer#4940]
I strictly follow the readme to use the same command to build the FFmpeg extension,but got errors below:

GEN	libavutil/libavutil.version
GEN	libavresample/libavresample.version
GEN	libavcodec/libavcodec.version
CC	libavcodec/ac3_parser.o
CC	libavcodec/adts_parser.o
CC	libavcodec/allcodecs.o
CC	libavcodec/arm/fft_fixed_init_arm.o
In file included from ./libavutil/channel_layout.h:25:0,
                 from libavcodec/ac3_parser.c:25:
/usr/local/andk/toolchains/arm-linux-androideabi-4.9/prebuilt/linux-x86_64/lib/gcc/arm-linux-androideabi/4.9.x/include/stdint.h:9:26: fatal error: stdint.h: No such file or directory
 # include_next <stdint.h>
                          ^
compilation terminated.
In file included from libavcodec/adts_parser.c:22:0:
/usr/local/andk/toolchains/arm-linux-androideabi-4.9/prebuilt/linux-x86_64/lib/gcc/arm-linux-androideabi/4.9.x/include/stdint.h:9:26: fatal error: stdint.h: No such file or directory
 # include_next <stdint.h>
                          ^
compilation terminated.
In file included from libavcodec/allcodecs.c:29:0:
libavcodec/avcodec.h:30:19: fatal error: errno.h: No such file or directory
 #include <errno.h>
                   ^
compilation terminated.
In file included from ./libavcodec/fft.h:33:0,
                 from libavcodec/arm/fft_fixed_init_arm.c:26:
/usr/local/andk/toolchains/arm-linux-androideabi-4.9/prebuilt/linux-x86_64/lib/gcc/arm-linux-androideabi/4.9.x/include/stdint.h:9:26: fatal error: stdint.h: No such file or directory
 # include_next <stdint.h>
                          ^

can anyone provide a help?

**Comment by tonihei** [comment]
As noted in the [README of the ffmpeg extension](https://github.com/google/ExoPlayer/blob/release-v2/extensions/ffmpeg/README.md), only NDK versions up to 15c are currrently supported. We are aware of this issue but haven't fixed it yet.

**Comment by tonihei** [comment]
Marking as an enhancement until this is fixed.

**Comment by AOrobator** [comment]
For anyone who stumbles upon this issue, it seems like

```
arm-linux-androideabi-gcc is unable to create an executable file.
C compiler test failed.
```

Is another symptom of this problem because gcc was removed in NDK r18b
https://developer.android.com/ndk/downloads/revision_history

**Comment by heitorri** [comment]
I have the same problem with android-ndk-r20

```
arm-linux-androideabi-gcc is unable to create an executable file.
C compiler test failed.
```

**Comment by ojw28** [comment]
This was fixed in 2.11.0.
